VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -122,30 +136,34 @@ public BaseViewHolder getTag(View convertView) {
     public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
     }
 
-    public void setVoteCountsEnabled(boolean voteCountsEnabled) {
-        this.voteCountsEnabled = voteCountsEnabled;
+    /**
```

Comment:
Maybe there should be a method definition as well, explaining the purpose/usecase of this utility method?